DependencyInjection/SonataMediaExtension.php

```diff
@@ -111,6 +117,24 @@ public function load(array $configs, ContainerBuilder $container)
 
         if (isset($bundles['SonataAdminBundle'])) {
             $loader->load(sprintf('%s_admin.xml', $config['db_driver']));
+
+            $sonataAdminConfig = $this->bundleConfigs['SonataAdminBundle'];
+
+            $sonataRoles = array();
+            if (isset($sonataAdminConfig['security']['role_admin'])) {
+                $sonataRoles[] = $sonataAdminConfig['security']['role_admin'];
+            } else {
+                $sonataRoles[] = 'ROLE_ADMIN';
+            }
```

Comment:
What about a `NEXT_MAJOR` statement to remove this `isset` check?

Reply:
It's ok if I insert the following?

>NEXT_MAJOR: remove the isset check and inject SonataAdminBundle role parameters instead of using prepend()

Reply:
Seems ok to me. @greg0ire WDYT?

Reply:
I think since now we regularly merge on master, we (and by we I mean @gremo) can take care of it as soon as it is merged, so this should be ok.

Reply:
@gremo Could you please do it?